index.html

```diff
@@ -3109,7 +3124,8 @@ <h2>Definition of Roles</h2>
 			<rdef>figure</rdef>
 			<div class="role-description">
 				<p>A perceivable <rref>section</rref> of content that typically contains a <a>graphical document</a>, images, code snippets, or example text. The parts of a <code>figure</code> MAY be user-navigable.</p>
-				<p>Authors SHOULD provide a reference to the <code>figure</code> from the main text, but the <code>figure</code> need not be displayed at the same location as the referencing element. Authors MAY reference text serving as a caption using <pref>aria-describedby</pref>. Authors MAY provide a label using <pref>aria-label</pref> or MAY reference text serving as a label using <pref>aria-labelledby</pref>.</p>
+				<p>Authors SHOULD provide a reference to the <code>figure</code> from the main text, but the <code>figure</code> need not be displayed at the same location as the referencing element. Authors MAY reference text serving as a caption using <pref>aria-describedby</pref>.</p>
+				<p>Authors MAY give each element with role figure a brief label that describes the purpose of the content in the figure.  Authors MAY use a visible caption as the first or last child element of the figure to provide a label.  The caption MAY be an instance of the standard host language <code>caption</code> element or an instance of an element with role <rref>caption</rref>.  If a visible label is present, but not contained in the first or last child element with a role of <code>caption</code>, authors SHOULD reference the visible label with <pref>aria-labelledby</pref>.</p>
```

Comment:
>The caption MAY be an instance of the standard host language <code>caption</code> element 

needs to be changed to
>The caption MAY be an instance of the standard host language <code>figcaption</code> element 
a `<caption>` outside of the context of a `<table>` would be invalid HTML.

This paragraph also gives leniency to where a `caption` may live in the DOM, while we also state it should be a child of the element it is providing the accessible name to.   if a `caption` can be place outside of a parent `table`, `figure`, `group` then should it not be indicated it should be associated via `aria-owns` to facilitate the expected parent/child relationship?

Reply:
the incorrect html element is still referenced in the latest update to the text.